### PR TITLE
 add 少数派专栏

### PR DESCRIPTION
**Pull request by LogicJake**
#1890

### CODE REVIEW

File: docs/other.md [hunk 1]
```diff
@@ -153,6 +153,8 @@
 
 <Route name="Matrix" author="feigaoxyz" example="/sspai/matrix" path="/sspai/matrix" />
 
+<Route name="专栏" author="LogicJake" example="/sspai/column/104" path="/sspai/column"  :paramsDesc="['专栏 id']"/>
```
Comment (DIYgod) [inline docs/other.md]: 应该是 `/sspai/column/:id`

File: lib/routes/sspai/column.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const axios = require('../../utils/axios');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const desApi = `https://sspai.com/api/v1/special_columns/${id}`;
+    let response = await axios.get(desApi);
```
Comment (DIYgod) [inline lib/routes/sspai/column.js]: 最好加上 Referer

Comment (LogicJake) [reply lib/routes/sspai/column.js]: done

File: docs/other.md (hunk 1 above)
Comment (LogicJake) [reply docs/other.md]: done

**Comment by DIYgod** [comment]
Merged, thanks!

**Pull request merged**